libvcx_core/src/api_vcx/api_handle/credential.rs

```diff
@@ -338,6 +338,11 @@ pub async fn decline_offer(handle: u32, connection_handle: u32, comment: Option<
     HANDLE_MAP.insert(handle, credential)
 }
 
+#[cfg(feature = "test_utils")]
+pub mod tests_utils {
+    pub const BAD_CREDENTIAL_OFFER: &str = r#"{"version": "0.1","to_did": "LtMgSjtFcyPwenK9SHCyb8","from_did": "LtMgSjtFcyPwenK9SHCyb8","claim": {"account_num": ["8BEaoLf8TBmK4BUyX8WWnA"],"name_on_account": ["Alice"]},"schema_seq_no": 48,"issuer_did": "Pd4fnFtRBcMKRVC2go5w3j","claim_name": "Account Certificate","claim_id": "3675417066","msg_ref_id": "ymy5nth"}"#;
+}
+
```

Comment:
Probably not in the scope of this PR, as this spans across most the crates in the repository, but we should clear out these test-only stuff from the actual code.

Instead, we could provide something like `common.rs` (or `common` as a dir) in the integration `tests` folder where we can declare all the tests related stuff. Then `mod common` and `use common::*` would allow us to use all of these in integration tests without having to intertwine these things with the code.

Alternatively, if these need to be available for unit/doc tests, we could have separate crates for them directly. It would make the implementation crates cleaner and smaller.

Reply:
Yes, the `test_utils` under the `/src` are used for unit tests - though I am actually leaning to keep this as is, I am not strongly opinionated. I find it nice that the unit test utils for some domain are in respective files and right next to the tests which are using them the most.

We can discuss on other channels further